src/plugin/plugin.cc

```diff
@@ -60,7 +61,9 @@ plugin::Zeek_Spicy::Plugin::Plugin() {
 
 void ::spicy::zeek::debug::do_log(const std::string& msg) {
     PLUGIN_DBG_LOG(*plugin::Zeek_Spicy::OurPlugin, "%s", msg.c_str());
-    HILTI_RT_DEBUG("zeek", msg);
+
+    if ( hilti::rt::isInitialized() )
+        HILTI_RT_DEBUG("zeek", msg);
```

Comment:
Is this required because of the changes over in Spicy? I am asking since in that case it would be much nicer if `HILTI_RT_DEBUG` handled checking its requirements internally instead of every user needing to think about it.

Reply:
Yes and no: The changes triggered it, but it was ill-defined to begin with: generally HILTI runtime functionality isn't usable before the library has been initialized. It just happened to work so far. That also addresses the 2nd part: I think it's reasonable for the macro to assume that the runtime has been initialized already (just like pretty much any other runtime functionality does).